compiler/dune-project

```diff
@@ -64,6 +64,15 @@
     (cmdliner (>= 1.0.2))
     (grain (>= 0))))
 
+(package
+  (name grainlsp)
+  (synopsis "Grain LSP")
+  (depends
+    (reason (>= 3.6.0))
+    (dune-build-info (>= 2.0))
+    (cmdliner (>= 1.0.2))
+    (grain (>= 0))))
```

Comment:
Missing a bunch of the deps that you included, basically anything in libraries and ppx's need to be here